src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -907,14 +932,11 @@
       min-width: $clr_baselineRem_4;
       border: none;
 
-      &.datagrid-fixed-column {
-        flex: 0 0 $clr-datagrid-fixed-column-size;
-      }
-
+      // TODO can width/column classes be combined? width is added programetically as part of rendering, column is
+      //  class in template
```

Comment:
I tried to consolidate these two classes but one is used in the datagrid renderer and the other is used in templates and it caused more problems than I anticipated.